streams/integration-tests/src/test/java/org/apache/kafka/streams/integration/EosIntegrationTest.java

```diff
@@ -1191,7 +1187,7 @@ private <K, V> void ensureCommittedRecordsInTopicPartition(final String topic,
             }
             if (tries >= maxTries) {
                 throw new AssertionError("No committed records in topic " + topic
-                    + ", partition " + partition + " after " + maxTries + " retries.");
+                        + ", partition " + partition + " after " + maxTries + " retries.");
```

Comment:
nit: avoid unnecessary reformatting

(might be some auto-formatting setting in your IDE -- I would recommend to disable auto formatting)

Reply:
I have checked that auto-formatting is off; the format setting might differ from current codestyle. Is there any doc to follow?

Reply:
Don't think we have docs. But it might be some IDE setting about 4 vs 8 whitespace indent...